packages/client/src/generation/utils.ts

```diff
@@ -207,7 +207,7 @@ export function getFieldType(field: DMMF.SchemaField): string {
 
 interface SelectReturnTypeOptions {
   name: string
-  actionName: ActionName
+  actionName: DMMF.ModelAction
```

Comment:
Thanks for the fix!
I think `ClientModelAction` would be a correct type here since it has to accept both `DMMF` and client-only actions